src/solid-forms.ts

```diff
@@ -1,2 +1,5 @@
-import { FormModel } from './classes/form-model';
-export { FormModel };
+import { FormModel } from './classes/form-model'
+import { ShexFormModel } from './classes/shex-form-model'
+import { formUi } from './utils'
```

Comment:
Can we make this an alias of `@utils` to be consistent with the rest of the repos?

Reply:
it's done